doc/classes/PoolByteArray.xml

```diff
@@ -134,6 +134,7 @@
 			</argument>
 			<description>
 				Sets the size of the array. If the array is grown, reserves elements at the end of the array. If the array is shrunk, truncates the array to the new size.
+				[b]Note:[/b] Added elements are not automatically initialized to 0 and may contain garbage.
```

Comment:
"garbage" sounds colloquial and thus not really suitable for documentation. Or is that the official name of uninitialized memory?

Reply:
I mean, its not really completelyc, see things such as "garbage collection", but in this case IMO "and may contain random values" or "previous values in memory" would be better.